internal/generator/vector/output/cloudwatch/cloudwatch.go

```diff
@@ -181,3 +186,19 @@ func ParseRoleArn(auth *obs.CloudwatchAuthentication, secrets observability.Secr
 	}
 	return ""
 }
+
+// ParseAssumeRoleArn search for matching valid assume role ARN
+func ParseAssumeRoleArn(auth *obs.CloudwatchAuthentication, secrets observability.Secrets) string {
```

Comment:
Some how I would expect this to be included in the ParseRoleArn function

Reply:
Yeah, ok, merging.